### DISCUSSION TITLE
Are there simple instructions to use ipse via UTM on my M1 Mac to install Windows 7?

### DISCUSSION THREAD

**Discussion by twdlup** [ipxe/ipxe#1035] [Q&A]
Hello! 

Brand new to trying to install Windows on my M1 Mac. I want to get into Windows 7 so I can help my elder aunt find and save files, favorites, etc, to migrate her to a Mac. Having to do this remote, so trying to run through the steps myself in Windows 7, also to see if can do any of this via RDP/remote to help as well.

I found the [UTM](https://mac.getutm.app) software. I downloaded Windows 7 via this Twitter [post](https://twitter.com/TheBobPony/status/1642869587942297600?t=hPh98m7NUsM47O3DMBKAhw&s=19)

I was able to get the ISO open on UTM, but I'm getting errors of "no bootable disk"... I can get the IPXE menu, but not sure how to just tell the thing to install Windows. 

**Thanks for any help!!!** 
<img width="888" alt="Screenshot 2023-09-03 at 8 00 26 PM" src="https://github.com/ipxe/ipxe/assets/132516634/dc103fea-3956-4faa-8e87-52b3e381198e">

**Comment by RulerOf** [comment]
This q&a page is for iPXE, which UTM uses for network boot support in its VMs, so isn't the right place to get support for the problem you're having.

That said, it's clear you're trying to install Win7 to a virtual hard disk by booting from the ISO image. Your problem is shown in the screenshot when the VM tried to boot from the ISO:

```
Boot failed: Could not read from CDROM (code 0004)
```

Check that your ISO image is not damaged, or seek out a different image file. If you're still having trouble, find the forum for UTM support and seek help there, as those folks will be better equipped to troubleshoot your problem.

**Comment by twdlup** [comment]
Hi @RulerOf! Thank you so much for getting back to me. I didn't realize what I had was a z7 archived ISO file. Once I unarchived it, so I had the actual ISO file (*face palm*), of course, it worked just fine. 

Should I move this conversation somewhere else? Or just delete it? I'm okay with whatever works. Thank you again!

**Reply by NiKiZe** [reply]
Good that you was able to find a solution.
Leave it as is. It might help someone else that tries to boot a ISO but get iPXE prompt, unaware of why it tries network boot.
